core/frontend/src/components/nmea-injector/NMEAInjector.vue

```diff
@@ -4,6 +4,19 @@
     elevation="0"
     class="mx-auto my-6 injectors-list"
   >
+  <v-card
+    class="mx-auto my-6"
+    width="500"
+    elevation="1"
+  >
+  <v-card-text>
+      <div class="text--primary">
+        The NMEA Injector receives NMEA via UDP or TCP and transforms it
+        into MAVLink data that is forwarded to the autopilot.
+        It currently supports the sentences GPGGA, GPRMC, GPGLL, and GPGNS.
```

Comment:
Can we add hyperlinks to http://aprs.gids.nl/nmea/#gga ?

`http://aprs.gids.nl/nmea/#{message}`

Reply:
I wish there was a more oficial source for these sentences...

Reply:
maybe here instead

Reply:
> I wish there was a more official source for these sentences...

Patrick's source views as less official/legit, but has more information. [This one](https://gpsd.gitlab.io/gpsd/NMEA.html#_gga_global_positioning_system_fix_data) seems somewhere in between?

That said, given NMEA is a proprietary standard the only actually "official" source we could provide is [the purchase link](https://www.nmea.org/nmea-0183.html). If an official source is a requirement then it's likely not worth including a link at all.

Perhaps also worth noting that people may be looking at this on a boat without internet access. There's some case to be made for links in BlueOS (wherever possible) to be internal, in which case perhaps we should be providing our own reference for specifically the messages we support.

Reply:
This same website is online for more than 20 years providing the same information, although official information is not available, that's not how open source works, we should trust people that provides it, having a source that we can point for users to checkout and understand is a better option.

> Perhaps also worth noting that people may be looking at this on a boat without internet access.

We have hyperlinks for online resources already being used in BlueOS.

> [This one](https://gpsd.gitlab.io/gpsd/NMEA.html#_gga_global_positioning_system_fix_data) seems somewhere in between?

The one that I'm referencing has a simple mark for each command, making the code to add the hyperlink super simple compared with the others.

```ts
function createLink(sentence: string): string {
    return `http://aprs.gids.nl/nmea/#{sentence.substring(2)}`
}
```

The others links will need to create an entire table to map the commands, making such feature not worth it codewise to maintain.

Reply:
> This same website is online for more than 20 years providing the same information, although official information is not available, that's not how open source works, we should trust people that provides it, having a source that we can point for users to checkout and understand is a better option.

I don’t doubt that it’s a good source - it just _looks_ old and cobbled together, which may make some users uncomfortable trusting it. 

Since the NMEA spec is frustratingly paywalled I’m ok with having that site linked to, or even having no link, but I do agree with Willian’s general sentiment that it’d be _nice_ if there was an official source we could just reference directly, instead of needing to link to something external that’s based on information scraped together from elsewhere.

Reply:
> This same website is online for more than 20 years providing the same information

that is part of the issue. it _feels_ like the website has been online for 20 years with no updates whatsoever

Reply:
I ended up using yours and a Trimble one, as neither supported all the sentences we do =|

Reply:
Lol, the resource I linked to and the Trimble one you ended up using both support all our current messages. The Trimble one even has generateable links like the aprs one 😂